### PR TITLE
set available JVM memory based on the current limit

### PR DESCRIPTION
**Pull request by aryszka**
spotio controller:

- [x] setting the memory for JVM
- [x] removing the VPA

Signed-off-by: Arpad Ryszka <arpad.ryszka@gmail.com>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

File: cluster/manifests/spotio-controller/vpa.yaml [hunk 1]
```diff
@@ -1,22 +0,0 @@
-{{ if and (index .Cluster.ConfigItems "spotio_account_id") (index .Cluster.ConfigItems "spotio_access_token") }}
```
Comment (linki) [inline cluster/manifests/spotio-controller/vpa.yaml:1]: Deletions require an entry in https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/deletions.yaml as well.

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**